chapter02/rwd/ex3-2.pl

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/perl
+
+print "Give me a radius: ";
+my $r = <>;
+my $pi = 3.141592654;
+
+print "The circunference is: ";
+if ($r > 0) {
+	print 2 * $pi * $r;
+} else {
+	print 0;
+}
+print "\n"
```

Comment:
Could have just had a newline on each of the previous print statement, instead of adding another line of code?